Source/Redemption/Public/HealthComponent.h

```diff
@@ -41,9 +44,18 @@ class REDEMPTION_API UHealthComponent : public UActorComponent
 	UPROPERTY(EditDefaultsOnly)
 	TSubclassOf<UUserWidget> GameOverWidgetClass;
 
+	UPROPERTY(BlueprintAssignable, Category = "Health")
+	FOnHealthLowDelegate OnHealthLow;
+
+	UPROPERTY(BlueprintAssignable, Category = "Health")
+	FOnHealthLowDelegate OnHealthRestored;
```

Comment:
are we using those delegates?

Reply:
No I guess that is a hold-over I forgot to delete.

Reply:
fair enough